### ISSUE TITLE
When refreshing, the width of the line number is incorrectly calculated.

### ISSUE THREAD

**Issue by allowing** [codemirror/CodeMirror#5620]
I found a bug.
There is no problem when loading for the first time. When refreshing, the width of the line number is incorrectly calculated.

![2018-10-16_192455](https://user-images.githubusercontent.com/20262949/47013461-cd468900-d179-11e8-870c-47290d7987c8.png)

**Comment by marijnh** [comment]
This is not enough information for us to reproduce the problem. Does it occur for you on the demos on the website? If not, can you set up a _minimal_ HTML demo that shows the issue?

**Comment by allowing** [comment]
This problem occurs when using the laravel framework's mix system.

**Comment by marijnh** [comment]
Right, but we're not going to test 3rd-party code, so if you want us to look at this, you'll have to figure out what that system is doing to cause the problem (most likely, it fails to call `refresh` on the editor after changing its size or style).

**Comment by allowing** [comment]
I have tested it many times and tried to load only `codemirror.js`, or this problem will occur. Estimated when `mix` is affected.
Since it is more troublesome to reproduce this problem, I uploaded some pictures for you to see. The Chinese network is too fuck. Waiting for half a day to visit github.

![2018-10-17_214032](https://user-images.githubusercontent.com/20262949/47092453-9778d180-d259-11e8-97c5-f3a28e9e00ec.png)
![2018-10-17_214052](https://user-images.githubusercontent.com/20262949/47092458-9ba4ef00-d259-11e8-802a-a88aa8171d3e.png)
![2018-10-17_214111](https://user-images.githubusercontent.com/20262949/47092461-9cd61c00-d259-11e8-8329-e00bdd876c2c.png)
![2018-10-17_214201](https://user-images.githubusercontent.com/20262949/47092467-9e074900-d259-11e8-80b3-75bbfa83f196.png)
